projects/infinityworks/github-exporter/Makefile

```diff
@@ -0,0 +1,68 @@
+REPO?=github-exporter
+CLONE_URL?=https://github.com/infinityworks/$(REPO).git
+GIT_COMMIT?=$(shell cat GIT_COMMIT)
+
+ifeq ("$(CLONE_URL)","")
+	$(error No clone url was provided.)
+endif
+
+ifeq ("$(REPO)","")
+	$(error No repository name was provided.)
+endif
+
+ifeq ("$(GIT_COMMIT)","")
+	$(error No git commit was provided.)
+endif
+
+DEVELOPMENT?=true
+AWS_ACCOUNT_ID?=$(shell aws sts get-caller-identity --query Account --output text)
+AWS_REGION?=us-west-2
+
+IMAGE_REPO?=$(AWS_ACCOUNT_ID).dkr.ecr.$(AWS_REGION).amazonaws.com
+IMAGE_NAME?=infinityworks/github-exporter
+# This tag is overwritten in the prow job to point to the commit hash
+IMAGE_TAG?=latest
+IMAGE?=$(IMAGE_REPO)/$(IMAGE_NAME):$(IMAGE_TAG)
+
+.PHONY: clone
+clone: clean
+	git clone $(CLONE_URL) $(REPO)
+	cd $(REPO) && git checkout $(GIT_COMMIT)
+	
+.PHONY: local-images
+local-images: clone
+	# Sleep to wait for buildkitd to start up
+	sleep 10
+	buildctl \
+		build \
+		--frontend dockerfile.v0 \
+		--opt platform=linux/amd64 \
+		--local dockerfile=./$(REPO) \
+		--local context=./$(REPO) \
+		--output type=oci,oci-mediatypes=true,name=$(IMAGE),dest=/tmp/github-exporter.tar
+
+.PHONY: images
+images: clone
+	# Sleep to wait for buildkitd to start up
+	sleep 10
```

Comment:
If we just started buildkitd somewhere, why aren't we waiting on that side?

Reply:
The `sleep` will not execute while the buildkit daemon is running, until after the daemon has stopped.

Reply:
Shouldn't the `clone` step take long enough to not require this sleep?

Reply:
The first time I ran it without the `sleep`, but it failed since the daemon hadn't started.
```bash
error: failed to get status: rpc error: code = Unavailable desc = connection error: desc = "transport: error while dialing: dial unix /run/buildkit/buildkitd.sock: connect: no such file or directory"
make: *** [local-images] Error 1
```
If the presubmit itself performs the clone operation so quickly, then we can expect it to fail for the postsubmit as well. Hence I have the `sleep` in both targets.